habitat/tasks/nav/nav_task.py

```diff
@@ -491,6 +510,15 @@ def reset_metric(self, episode):
             self._map_resolution,
         )
         self._previous_xy_location = (a_y, a_x)
+        # draw optimal path
+        maps.draw_path(
```

Comment:
@mathfac @abhiskk Thoughts on draw order?  Atm the optimal path will be drawn over the agents path

Reply:
Not sure what you mean by this. Draw_path will only happen in `reset_metric()`. Agents path will be drawn step by step in `update_metric()`-> `update_map()`, over the optimal path.

Reply:
Seems like I miss understood!